syft/pkg/package_test.go

```diff
@@ -264,10 +250,6 @@ func TestPackage_Merge(t *testing.T) {
 				Locations: source.NewLocationSet(
 					originalLocation,
 				),
-				Licenses: []string{
```

Comment:
should have representation of licenses in the merge test

Reply:
Added merge packages license test here